pulsar-broker/src/main/java/org/apache/pulsar/broker/service/PulsarStats.java

```diff
@@ -136,9 +136,6 @@ public synchronized void updateStats(
                                 } catch (Exception e) {
                                     log.error("Failed to generate topic stats for topic {}: {}", name, e.getMessage(), e);
                                 }
-                                // this task: helps to activate inactive-backlog-cursors which have caught up and
-                                // connected, also deactivate active-backlog-cursors which has backlog
-                                ((PersistentTopic) topic).getManagedLedger().checkBackloggedCursors();
```

Comment:
if we have removed from here then who triggers `checkBackloggedCursors(..)`?

Reply:
uhm.. :)

Reply:
Fixed